src/SymbolCollector.Console/ConsoleUploader.cs

```diff
@@ -29,13 +29,6 @@ public async Task StartUploadSymbols(IEnumerable<string> paths, string bundleId,
         SentrySdk.ConfigureScope(s =>
         {
             s.Transaction = transaction;
-            s.AddEventProcessor(@event =>
```

Comment:
Why remove this though? it's helpful to see the summary at the time of the event